Velaptor/OpenGL/GPUData/RectGPUData.cs

```diff
@@ -0,0 +1,107 @@
+
+        /// <summary>
+        /// Initializes static members of the <see cref="RectGPUData"/> struct.
+        /// </summary>
+        static RectGPUData() => Stride = RectVertexData.GetTotalBytes() * TotalVertexItems;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="RectGPUData"/> struct.
+        /// </summary>
+        /// <param name="vertex1">The first vertex of the rectangle.</param>
+        /// <param name="vertex2">The second vertex of the rectangle.</param>
+        /// <param name="vertex3">The third vertex of the rectangle.</param>
+        /// <param name="vertex4">The fourth vertex of the rectangle.</param>
+        public RectGPUData(RectVertexData vertex1, RectVertexData vertex2, RectVertexData vertex3, RectVertexData vertex4)
+        {
+            Vertex1 = vertex1;
+            Vertex2 = vertex2;
+            Vertex3 = vertex3;
+            Vertex4 = vertex4;
+        }
+
+        /// <summary>
+        /// Gets the vertex data for the top left vertex of a rectangle.
+        /// </summary>
+        /// <remarks>
+        ///     This is first vertex of the top left triangle that makes up the rectangle.
+        /// </remarks>
+        public RectVertexData Vertex1 { get; }
+
+        /// <summary>
+        /// Gets the vertex data for the bottom left vertex of a rectangle.
+        /// </summary>
+        /// <remarks>
+        /// <para>
+        ///     This is second vertex of the bottom left triangle that makes up the rectangle.
+        /// </para>
+        /// <para>
+        ///     This vertex is shared with the second vertex of the bottom right triangle of the rectangle.
+        /// </para>
+        /// </remarks>
+        public RectVertexData Vertex2 { get; }
+
+        /// <summary>
+        /// Gets the vertex data for the top right vertex of a rectangle.
+        /// </summary>
+        /// <remarks>
+        /// <para>
+        ///     This is third vertex of the top right triangle that makes up the rectangle.
+        /// </para>
+        /// <para>
+        ///     This vertex is shared with the first vertex of the bottom right triangle of the rectangle.
+        /// </para>
+        /// </remarks>
+        public RectVertexData Vertex3 { get; }
+
+        /// <summary>
+        /// Gets the vertex data for the bottom right vertex of a rectangle.
+        /// </summary>
+        /// <remarks>
+        ///     This is the third vertex of the bottom right triangle of the rectangle.
+        /// </remarks>
+        public RectVertexData Vertex4 { get; }
+
+        /// <summary>
+        /// Creates an empty instance of <see cref="RectGPUData"/>.
+        /// </summary>
+        /// <returns>An empty instance.</returns>
+        public static RectGPUData Empty() => new (RectVertexData.Empty(), RectVertexData.Empty(), RectVertexData.Empty(), RectVertexData.Empty());
+
+        /// <summary>
+        /// The total number of bytes that the GPU data contains.
```

Comment:
Refactor this to below:

```suggestion
        /// The total number of bytes that the <see cref="RectGPUDta"/>  data contains.
```